cypher/cypher-docs/src/docs/dev/ql/administration/constraints/index.asciidoc

```diff
@@ -252,9 +252,70 @@ include::drop-a-constraint.asciidoc[leveloffset=+2]
 
 include::drop-a-non-existing-constraint.asciidoc[leveloffset=+2]
 
+
```

Comment:
not sure why we need/want 2 empty lines here

Reply:
No, one should be enough.